nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyFacet.java

```diff
@@ -184,7 +184,9 @@ private Component getOrCreateComponent(final StorageTx tx,
       component = tx.createComponent(bucket, getRepository().getFormat())
           .group(coords.getGroup())
           .name(coords.getProject())
-          .version(coords.getVersion());
+          .version(String.format("%s-%s", coords.getVersion(), coords.getChannel()));
+      component.formatAttributes().set("baseVersion", coords.getVersion());
```

Comment:
Please create new one method called getComponentVersion in ConanProxyHelper, because this logic use in 2 places

Reply:
Fixed 6493481cc0fc0d14e7d56fcee608ad892add7f2b